styleguide/_themes/derek/scss/patterns/grid/productFeatures.scss

```diff
@@ -30,11 +31,11 @@
 }
 
 @media only screen and (max-width: $screen-sm) {
-  .features {
+  .features-container {
     display: block;
     margin-top: 10px;
 
-    .desc {
+    .features-desc {
```

Comment:
Can we add the `make-md-column()` here and get rid of the column classes while we're at it?